t/01-style.t

```diff
@@ -22,6 +22,9 @@ is qx{git grep -I -l '^use Test::\\(Exception\\|Fatal\\)' t/**.t}, '',
 is qx{git grep -I -l '^\\(throws\\|dies\\|lives\\)_ok.*\<sub\>' t/**.t}, '',
   'Only use simplified prototyped Test::Exception functions';
 is qx{git grep -I -l 'like.*\$\@' t/**.t}, '', 'Use throws_ok instead of manual checks of exceptions';
+# https://progress.opensuse.org/issues/176862 - It should be enaled once all modules have Feature::Compat::Try
+#is qx{git grep -I -l '^use \\(Try::Tiny\\|TryCatch\\)' lib/**.pm script/** t/**.t}, '',
+#  'No Try::Tiny or TryCatch necessary, use Feature::Compat::Try and later native Perl';
```

Comment:
As you didn't include all necessary changes in this PR anymore as we discussed this change should not be part of this PR but kept in a separate branch that you rebase on top